### PR TITLE
Added Firefox Example to Ruby

### PR DESCRIPTION
**Pull request by spider-sauce**
Also changed `platform_version` to `platform_name` per the changes made to the hash in the docs: https://www.rubydoc.info/gems/selenium-webdriver/Selenium%2FWebDriver%2FRemote%2FW3C%2FCapabilities:initialize

### CODE REVIEW

File: ruby/rspec/chrome_example_spec.rb [hunk 1]
```diff
@@ -24,9 +24,14 @@
...
+        platform_name: 'windows 10',
         browser_version: '61.0',
-        "goog:chromeOptions": {w3c: true},
+        # w3c is note enabled by default in Chrome, set the w3c setting to true
+        "goog:chromeOptions" => {w3c: true},
+        # use sauce:options to handle all saucelabs.com-specific capabilities such as:
+        # username, accesskey, build number, test name, timeouts etc.
         "sauce:options" => {
             name: test.full_description,
             seleniumVersion: '3.11.0',
```
Comment (mmerrell-sauce) [inline ruby/rspec/chrome_example_spec.rb]: I tested with the latest version of Selenium (3.141.59), and it seems to work. I think we should try to reflect that version in all these examples.

File: ruby/rspec/chrome_example_spec.rb [hunk 2]
```diff
@@ -24,9 +24,14 @@
   before(:each) do |test|
     caps = {
         browser_name: 'chrome',
-        platform_version: 'Windows 10',
+        # platform_version now becomes platform_name with the W3C bridge.
+        # For more details see: https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/Remote/W3C/Capabilities
+        platform_name: 'windows 10',
         browser_version: '61.0',
```
Comment (mmerrell-sauce) [inline ruby/rspec/chrome_example_spec.rb]: It looks like Chrome is up to version 71.0, which works fine according to some tests I've run.

File: ruby/rspec/chrome_example_spec.rb (hunk 1 above)
Comment (spider-sauce) [reply ruby/rspec/chrome_example_spec.rb]: Updated Selenium Version to 3.141.59 and tested. Still runs!

File: ruby/rspec/chrome_example_spec.rb (hunk 2 above)
Comment (spider-sauce) [reply ruby/rspec/chrome_example_spec.rb]: My logic was to use the minimum version in order to indicate that W3C works from that version up. For the next push I'll update the browsers to their latest versions if that's what we decide.

**Comment by titusfortner** [comment]
1. We want to encourage the latest version of Selenium in general because that's where the best support is for the latest browsers.
2. If we use the latest browsers we won't have to update this code as often. :)

**Pull request merged**